haystack_experimental/dataclasses/tool.py

```diff
@@ -196,6 +207,85 @@ def get_weather(
 
         return Tool(name=function.__name__, description=tool_description, parameters=schema, function=function)
 
+    @classmethod
+    def from_openapi_spec(cls, spec: Union[str, Path], **kwargs: OpenAPIKwargs) -> List["Tool"]:
```

Comment:
Differently from `@from_function` class method, this method returns a list of Tools. 
Calling `Tool.from_openapi_spec` and getting a list of Tools seems a bit confusing  from a semantic point of view.

Maybe is it better to create a function (not a class method): something like `create_tools_from_openapi_spec`.
WDYT? Do you have better ideas?